validator.js

```diff
@@ -45,6 +47,7 @@ export const validateBom = (bomJson) => {
     },
   });
   addFormats(ajv);
+  addFormatsDraft2019(ajv);
```

Comment:
This is not required when we use validate-iri.

Reply:
This is required when strict is true